app/experimenter/experiments/tests/test_views.py

```diff
@@ -298,6 +298,164 @@ def test_list_view_lists_experiments_with_default_order_no_archived(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(list(context["experiments"]), list(experiments))
 
+    def set_up_date_tests(self):
+
+        self.exp_1 = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT,
+            name="experiment 1",
+            proposed_start_date=datetime.date(2019, 4, 5),
+            proposed_duration=30,
+            proposed_enrollment=3,
+        )
+
+        self.exp_2 = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT,
+            name="experiment 2",
+            proposed_start_date=datetime.date(2019, 3, 29),
+            proposed_duration=14,
+            proposed_enrollment=4,
+        )
+
+        self.exp_3 = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT,
+            name="experiment 3",
+            proposed_start_date=datetime.date(2019, 5, 29),
+            proposed_duration=30,
+            proposed_enrollment=None,
+        )
+
+        self.exp_4 = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT,
+            name="experiment 4",
+            proposed_start_date=datetime.date(2019, 3, 15),
+            proposed_duration=25,
+            proposed_enrollment=1,
+        )
+
+        self.start_range_date = "2019-04-01"
+        self.end_range_date = "2019-05-01"
+        self.user_email = "user@example.com"
+
+    def test_list_shows_all_experiments_with_start_in_range(self):
+
+        self.set_up_date_tests()
+
+        response = self.client.get(
+            "{url}?{params}".format(
+                url=reverse("home"),
+                params=urlencode(
+                    {
+                        "experiment_date_field": "starts_between",
```

Comment:
After you move these values to constants remember to refer to constants here too